plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/catalog/nessie/IcebergNessieCatalogModule.java

```diff
@@ -48,7 +48,6 @@ public static NessieIcebergClient createNessieIcebergClient(IcebergNessieCatalog
         return new NessieIcebergClient(
                 HttpClientBuilder.builder()
```

Comment:
HttpClientBuilder is deprecated now. Let's migrate to NessieClientBuilder in follow-up.

Reply:
ACK